test/ui/CatetgoryFacet/CategoryFacetTest.ts

```diff
@@ -0,0 +1,14 @@
+import { CategoryFacet } from '../../../src/ui/CategoryFacet/CategoryFacet';
+import * as Mock from '../../MockEnvironment';
+
+export function CategoryFacetTest() {
+  describe('CategoryFacet', () => {
+    it('get children calls get children on the CategoryValueRoot', () => {
```

Comment:
I think it would help the test readability if `getChildren` did not have a space.

E.g. `it('calling getChildren calls getChildren on the CategoryValueRoot', () => {...})`

Also, how is the check on CategoryValueRoot performed? Is `.cmp` the Root? If so, it would help readability to store it in a variable.